### DISCUSSION TITLE
Scroller auto resize 时能否让用户无感知

### DISCUSSION THREAD

**Discussion by haohaio** [antvis/X6#3214] [General]
我看 Scroller 代码中 onUpdate 时，出于性能考虑使用了防抖函数 `this.onUpdate = FunctionExt.debounce(this.onUpdate, 200)`，但这样用户体验并不是很好。从用户体验角度来说，能否去掉这个优化呢？

**Comment by NewByVector** [comment]
之间增加防抖是处于性能考虑的，这块还需要考虑一下，我先提一个 issue 跟进一下。

**Reply by NewByVector** [reply]
https://github.com/antvis/X6/issues/3223

**Reply by hustcc** [reply]
@haohaio 有兴趣来 PR 解决吗？
